README.md

```diff
@@ -73,3 +73,21 @@ The App will attempt to load the trust policy from
 `.github/chainguard/${NAME}.sts.yaml` from `${REPO}` and if the provided `${TOKEN}`
 satisfies those rules, it will return a token with the permissions in the trust
 policy.
+
+### Release cadence
+
+Our release cadence at this moment is set to when is needed, meaning if we have a bug fix or a new feature
+we will migth make a new release.
+
+### Permission updates
+
+Sometimes we need to add or remove a GitHub Permission in order to add/remove permissions that will be include in the
+octo-sts token for the users.
```

Comment:
```suggestion
Sometimes we need to add or remove a GitHub Permission in order to add/remove permissions that will be include in the
octo-sts token for the users. Due to the nature of GitHub Apps, OctoSTS must request all permissions it might need to use, even if you don't want to use them for your particular installation or policy.
```